src/withHTMLSubtitles/withHTMLSubtitles.js

```diff
@@ -51,9 +51,19 @@ function withHTMLSubtitles(Video) {
         var size = 100;
         var offset = 0;
         var textColor = 'rgb(255, 255, 255)';
-        var backgroundColor = 'rgba(0, 0, 0, 0)';
+
+        var subsBgStyle = 1;
+        // Calculate the #c010r. First slice the # and cast to int
+        var intColor = parseInt(outlineColor.slice(1), 16);
+        var backgroundColor = 'rgba(' +
```

Comment:
What is the purpose of this?

Reply:
I took this from the stremio 4 desktop version. 

I believe it's meant to put some background which is based on the outline colour passed.